components/jupyter-web-app/frontend/src/app/resource-form/volume/volume.component.ts

```diff
@@ -58,6 +63,14 @@ export class VolumeComponent implements OnInit, OnDestroy {
     }
   }
 
+// ----- onChange of the New / Existing Volume -----
+  selectType(event): void {
+    this.typeSelected = event.value;
+    if (this.typeSelected === 'New') {
+      this.volume.controls.name.setValue(this.currentVolName);
+    }
```

Comment:
*nit*: Prevent unnecessary code depth
```suggestion
    if (this.typeSelected !== 'New') return;
    this.volume.controls.name.setValue(this.currentVolName);
```

Reply:
Will test this and commit again

Reply:
Updated and submitted the commit